modules/installation-initializing-manual.adoc

```diff
@@ -111,6 +115,53 @@ endif::restricted[]
 ====
 For some platform types, you can alternatively run `./openshift-install create install-config --dir=<installation_directory>` to generate an `install-config.yaml` file. You can provide details about your cluster configuration at the prompts.
 ====
+ifdef::ash[]
++
+Make the following modifications for Azure Stack Hub:
+
+.. Configure the cluster to allow for manually managing cloud credentials:
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Manual <1>
+compute:
+----
+<1> Set the `credentialsMode` parameter to `Manual`.
```

Comment:
This is fine. It is also optional. Manual is the only credentialsMode for Azure Stack Hub so if this is omitted we automatically set the credntialsMode to Manual. If they select another mode, the installer will catch it with validation.

So it's fine leaving this in if we want to be thorough or we can omit if we want to simplify.

Reply:
I'll just omit it. No need in calling out something that is done automatically :+1: